src/GameManager.java

```diff
@@ -0,0 +1,26 @@
+import java.util.List;
+
+public class GameManager {
+    private final Game game;
+
+    public GameManager(Game game) {
+        this.game = game;
+    }
+
+    public Game getGame() {
+        return this.game;
+    }
+
+    public void play() {
+        while (this.game.getGameWon() == false) {
+            this.game.playGame();
+            int playersWon = 0;
+            List<Player> players = this.game.getBoard().getPlayerList();
+            for(Player player : players) {
+                playersWon += (player.getHasWon())?1:0;
```

Comment:
should not ask player if they've won. Game should know about it

Reply:
No, it basically checks if the present player from the player list has won by checking "getHasWon" function. But the function name could be made a bit better I agree, if that's your point.

Reply:
The game should have a logic to determine if the present player has won. The player shouldn't.

Reply:
That logic is being implemented in line 107-112 of Game.java. And accordingly it is being set for the present player whether he has won by player.setHasWon(). Are you suggesting that I make the code a little modular and move that logic out of playGame()?

Reply:
I'm suggesting to remove hasWon from Player.